test/dbtvault_test/dbt_project.yml

```diff
@@ -18,6 +18,8 @@ clean-targets:
 
 vars:
   max_date: TO_DATE("9999-12-31")
```

Comment:
Is max date still used anywhere?

Reply:
That variable has always been defined there yet I don't think it has ever been referenced anywhere. Do you want me to remove it?

Reply:
Please double check first by doing a project wide search (PyCharm Ctrl+Shift+F, click project, also click the W to make it match only max_date and not max_datetime). Then yes, please delete it if it's not found anywhere.

Reply:
OK, searched, not found any usage, removed, committed, pushed.